### DISCUSSION TITLE
exporting preferences

### DISCUSSION THREAD

**Discussion by olbotta** [xournalpp/xournalpp#3699] [Q&A]
is there a way to export (and import) preferences (and toolbar layouts)? i cannot find one
(v1.1.0 on Ubuntu 20.04)

**edit**: it would also be nice to be able to export the toolbar custom layout. this is not possible copy-pasting the `settings.xml` file

**Comment by rolandlo** [comment]
There is not a direct way within Xournal++. You can save the `settings.xml` file though and reuse it (although switching between versions or using it with different devices may also cause trouble). It is located in the config folder. See [file locations](https://xournalpp.github.io/guide/file-locations/).

**Reply by olbotta** [reply]
thank you! i noticed the settings file, it is XML so in theory is backward-compatible.
it would be nice to have an "official" (and so foolproof) way to export and import this file.
